public/html/base.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Form Hub</title>
+    <link rel="stylesheet" href="../css/index.css">
```

Comment:
Давайте пути до файлов писать абсолютными от рута вашего проекта. Так проще понимать где находится в проекте файл который вы импортируете. В данном случае
```suggestion
    <link rel="stylesheet" href="/css/index.css">
```
Исправьте во всех остальных местах тоже